v2.8.2 "--no-track" option throws unhanded exception

### What happened ?

Didn't notice this before v2.8.2, so assuming this is a regression in the newest version.

The error log is included below:

```
[INFO] Checking url = https://mangadex.org/chapter/1e864547-08c0-405a-b465-1c6d32dfe7bd
[INFO] Found chapter None from manga "The girl I was interested in was a Two-Mouthed Youkai."
[INFO] Download directory is set to "/Users/scheris/Downloads/The girl I was interested in was a Two-Mouthed Youkai"
[INFO] Using raw format
[ERROR] Unhandled exception, AttributeError: 'NoneType' object has no attribute 'images'
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/cli/__init__.py", line 73, in _main
    download(args)
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/cli/download.py", line 33, in download
    url(args, args.type)
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/cli/url.py", line 161, in __call__
    self.func(self.id, args)
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/cli/url.py", line 116, in download_chapter
    dl_chapter(*args)
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/main.py", line 206, in download_chapter
    fmt.main()
  File "/opt/homebrew/lib/python3.11/site-packages/mangadex_downloader/format/raw.py", line 59, in main
    for im_info in file_info.images:
                   ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'images'
```

### What did you expect to happen ?

Should have downloaded normally, like if I remove `--no-track` and it works as expected.

### OS version

macOS 13.2.1 (M1)

### App version

2.8.2

### Installation origin

PyPI (Python Package Index)

### Installation origin (other sources)

_No response_

### Reproducible command

mangadex-dl --no-track --use-compressed-image https://mangadex.org/chapter/1e864547-08c0-405a-b465-1c6d32dfe7bd

### Additional context

_No response_

--- a/mangadex_downloader/format/base.py
+++ b/mangadex_downloader/format/base.py
@@ -277,13 +277,20 @@ class BaseFormat:
 
     def get_new_chapters(self, file_info, chapters, name, log_output=True):
         """Retrieve new chapters for volume and single formats"""
+        if file_info is None:
+            fi_chapters = []
+            fi_completed = False
+        else:
+            fi_chapters = file_info.chapters
+            fi_completed = file_info.completed
+
         # Check for new chapters in volume
         new_chapters = []
         for chap_class, _ in chapters:
-            if chap_class.id not in file_info.chapters:
+            if chap_class.id not in fi_chapters:
                 new_chapters.append(chap_class.name)
 
-        if new_chapters and file_info.completed and log_output:
+        if new_chapters and fi_completed and log_output:
             log.info(
                 f"There is new {len(new_chapters)} chapters in {name}. " \
                 f"Re-downloading {name}..."

--- a/mangadex_downloader/format/raw.py
+++ b/mangadex_downloader/format/raw.py
@@ -41,7 +41,6 @@ class Raw(BaseFormat):
     def main(self):
         base_path = self.path
         manga = self.manga
-
         self.write_tachiyomi_info()
 
         # Recreate DownloadTracker JSON file if --replace is present
@@ -56,12 +55,19 @@ class Raw(BaseFormat):
             file_info = self.get_fi_chapter_fmt(chap_name, chap_class.id)
             chapter_path = create_directory(chap_name, base_path)
 
-            for im_info in file_info.images:
+            if file_info is None:
+                fi_images = []
+                fi_completed = False
+            else:
+                fi_images = file_info.images
+                fi_completed = file_info.completed
+
+            for im_info in fi_images:
                 verified = verify_sha256(im_info.hash, chapter_path / im_info.name)
                 if not verified:
                     failed_images.append(im_info)
                 
-            if failed_images and file_info.completed:
+            if failed_images and fi_completed:
                 log.warning(
                     f"Found {len(failed_images)} unverified or missing images from {chap_name}. " \
                     "Re-downloading..."
@@ -73,7 +79,7 @@ class Raw(BaseFormat):
 
                     log.debug(f"Removing unverified image '{im_path.resolve()}'")
                     delete_file(im_path)
-            elif file_info.completed:
+            elif fi_completed:
                 log.info(f"'{chap_name}' is verified. no need to re-download")
                 self.mark_read_chapter(chap_class)
                 continue
@@ -129,6 +135,13 @@ class RawVolume(BaseFormat):
             file_info = self.get_fi_volume_or_single_fmt(volume_name)
             new_chapters = self.get_new_chapters(file_info, chapters, volume_name)
 
+            if file_info is None:
+                fi_images = []
+                fi_completed = False
+            else:
+                fi_images = file_info.images
+                fi_completed = file_info.completed
+
             # Create volume cover
             if self.config.use_volume_cover:
                 img_name = count.get() + ".png"
@@ -138,17 +151,17 @@ class RawVolume(BaseFormat):
                 count.increase()
 
             # Only checks if ``file_info.complete`` state is True
-            if new_chapters and file_info.completed:
+            if new_chapters and fi_completed:
                 # Re-create directory to prevent error
                 shutil.rmtree(volume_path, ignore_errors=True)
                 volume_path = create_directory(volume_name, base_path)
 
-            for im_info in file_info.images:
+            for im_info in fi_images:
                 verified = verify_sha256(im_info.hash, volume_path / im_info.name)
                 if not verified:
                     failed_images.append(im_info)
                 
-            if failed_images and file_info.completed and not new_chapters:
+            if failed_images and fi_completed and not new_chapters:
                 log.warning(
                     f"Found {len(failed_images)} unverified or missing images from {volume_name}. " \
                     "Re-downloading..."
@@ -160,7 +173,7 @@ class RawVolume(BaseFormat):
 
                     log.debug(f"Removing unverified image '{im_path.resolve()}'")
                     delete_file(im_path)
-            elif file_info.completed:
+            elif fi_completed:
                 log.info(f"'{volume_name}' is verified. no need to re-download")
                 self.mark_read_chapter(*chapters)
                 continue
@@ -170,7 +183,7 @@ class RawVolume(BaseFormat):
             chapter_failed_images = set(i.chapter_id for i in failed_images)
 
             for chap_class, images in chapters:
-                if chap_class.id not in chapter_failed_images and file_info.completed:
+                if chap_class.id not in chapter_failed_images and fi_completed:
                     count.increase(chap_class.pages)
                     continue
 
@@ -236,18 +249,25 @@ class RawSingle(BaseFormat):
         file_info = self.get_fi_volume_or_single_fmt(name)
         new_chapters = self.get_new_chapters(file_info, cache, name)
 
+        if file_info is None:
+            fi_images = []
+            fi_completed = False
+        else:
+            fi_images = file_info.images
+            fi_completed = file_info.completed
+
         # Only checks if ``file_info.complete`` state is True
-        if new_chapters and file_info.completed:
+        if new_chapters and fi_completed:
             # Re-create directory to prevent error
             shutil.rmtree(path, ignore_errors=True)
             path = create_directory(name, base_path)
 
-        for im_info in file_info.images:
+        for im_info in fi_images:
             verified = verify_sha256(im_info.hash, path / im_info.name)
             if not verified:
                 failed_images.append(im_info)
             
-        if failed_images and file_info.completed and not new_chapters:
+        if failed_images and fi_completed and not new_chapters:
             log.warning(
                 f"Found {len(failed_images)} unverified or missing images from {name}. " \
                 "Re-downloading..."
@@ -259,7 +279,7 @@ class RawSingle(BaseFormat):
 
                 log.debug(f"Removing unverified image '{im_path.resolve()}'")
                 delete_file(im_path)
-        elif file_info.completed:
+        elif fi_completed:
             log.info(f"'{name}' is verified. no need to re-download")
             self.mark_read_chapter(*cache)
 
@@ -272,7 +292,7 @@ class RawSingle(BaseFormat):
         chapter_failed_images = [i.chapter_id for i in failed_images]
 
         for chap_class, images in cache:
-            if chap_class.id not in chapter_failed_images and file_info.completed:
+            if chap_class.id not in chapter_failed_images and fi_completed:
                 count.increase(chap_class.pages)
                 continue
 

--- a/mangadex_downloader/tracker/info_data/sqlite.py
+++ b/mangadex_downloader/tracker/info_data/sqlite.py
@@ -91,6 +91,11 @@ class FileInfo:
     images: Union[None, List[ImageInfo]]
     chapters: Union[None, List[ChapterInfo]]
 
+    @classmethod
+    def dummy(cls):
+        """Create dummy FileInfo for all formats if --no-track is used"""
+        return cls()
+
     def __post_init__(self):
         if self.images is not None:
             self.images = [ImageInfo(*(i[0], i[1], i[2])) for i in self.images]

